### ISSUE TITLE
conda update spyder

### ISSUE THREAD

**Issue by analog71** [spyder-ide/spyder#23356]
### What steps will reproduce the problem?

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    certifi-2024.12.14         |   py39haa95532_0         161 KB
    spyder-5.2.2               |   py39haa95532_1         9.4 MB
    ------------------------------------------------------------
                                           Total:         9.5 MB

The following packages will be UPDATED:

  ca-certificates                       2024.7.2-haa95532_0 --> 2024.11.26-haa95532_0
  certifi                           2024.7.4-py39haa95532_0 --> 2024.12.14-py39haa95532_0

The following packages will be SUPERSEDED by a higher-priority channel:

  spyder             conda-forge::spyder-5.2.2-py39hcbf530~ --> pkgs/main::spyder-5.2.2-py39haa95532_1

Proceed ([y]/n)? y

Downloading and Extracting Packages:

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\antho\miniconda3\envs\python_dsp_new\lib\site-packages\spyder\plugins\maininterpreter\widgets\status.py", line 142, in update_envs
    self.envs.update(**output)
TypeError: dict.update() argument after ** must be a mapping, not NoneType
Traceback (most recent call last):
  File "C:\Users\antho\miniconda3\envs\python_dsp_new\lib\site-packages\spyder\plugins\maininterpreter\widgets\status.py", line 142, in update_envs
    self.envs.update(**output)
TypeError: dict.update() argument after ** must be a mapping, not NoneType
Traceback (most recent call last):
  File "C:\Users\antho\miniconda3\envs\python_dsp_new\lib\site-packages\spyder\plugins\maininterpreter\widgets\status.py", line 142, in update_envs
    self.envs.update(**output)
TypeError: dict.update() argument after ** must be a mapping, not NoneType
Traceback (most recent call last):
  File "C:\Users\antho\miniconda3\envs\python_dsp_new\lib\site-packages\spyder\plugins\maininterpreter\widgets\status.py", line 142, in update_envs
    self.envs.update(**output)
TypeError: dict.update() argument after ** must be a mapping, not NoneType
```

## Versions

* Spyder version: 5.2.2 
* Python version: 3.9.13
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.1 (OK)
chardet >=2.0.0               :  5.0.0 (OK)
cloudpickle >=0.5.0           :  2.2.0 (OK)
cookiecutter >=1.6.0          :  2.1.1 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.6.0;<8.0.0        :  7.33.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.16.0 (OK)
keyring >=17.0.0              :  23.9.3 (OK)
nbconvert >=4.0               :  7.2.3 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
paramiko >=2.4.0              :  2.11.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.3 (OK)
pygments >=2.0                :  2.13.0 (OK)
pylint >=2.5.0                :  2.15.5 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.1.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.2 (OK)
qtawesome >=1.0.2             :  1.2.1 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.2.1 (OK)
rtree >=0.9.7                 :  1.0.1 (OK)
setuptools >=49.6.0           :  65.5.0 (OK)
sphinx >=0.6.6                :  5.3.0 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.5.0 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.9 (OK)
zmq >=17                      :  24.0.1 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  3.6.1 (OK)
numpy >=1.7                   :  1.23.4 (OK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  1.9.3 (OK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by ccordoba12** [comment]
Hey @analog71, thanks for reporting. Please run instead

    conda install spyder=5.4

That should allow you to update Spyder to a version that solves the problem you're experiencing,

Let us know if it works for you.

**Comment by analog71** [comment]
installed 5.4.1 and issue still there.
Traceback (most recent call last):
  File "C:\Users\antho\miniconda3\envs\py_test\lib\site-packages\spyder\plugins\maininterpreter\widgets\status.py", line 177, in update_envs
    self.envs.update(**output)
TypeError: dict.update() argument after ** must be a mapping, not NoneType

* Spyder version: 5.4.1  (conda)
* Python version: 3.9.16 64-bit
* Qt version: 5.15.8
* PyQt5 version: 5.15.9
* Operating System: Windows 10

**Comment by ccordoba12** [comment]
The issue was solved in 5.4.3. I thought that version was available in Anaconda.

Then you need to run

    conda install spyder=5.5

**Comment by analog71** [comment]
fixed
